sdk/search/Azure.Search.Documents/src/Utilities/Constants.cs

```diff
@@ -95,30 +94,23 @@ internal static class Constants
         /// <summary>
         /// The value key.
         /// </summary>
-        public const string ValueKey = "value";
+        public static readonly JsonEncodedText ValueKeyJson = JsonEncodedText.Encode("value");
 
         /// <summary>
         /// The count key.
         /// </summary>
-        public const string CountKey = "count";
-
-        /// <summary>
-        /// Initial ArrayPool rental size for copying unseekable streams in
-        /// our sync <see cref="JsonExtensions.Deserialize{T}"/> method.
-        /// </summary>
-        public const int UnseekableStreamInitialRentSize = 4096;
-
-        /// <summary>
-        /// Byte order mark for a UTF8 document used by our sync
-        /// <see cref="JsonExtensions.Deserialize{T}"/> method.
-        /// </summary>
-        public static ReadOnlySpan<byte> Utf8Bom => new byte[] { 0xEF, 0xBB, 0xBF };
+        public static readonly JsonEncodedText CountKeyJson = JsonEncodedText.Encode("count");
 
         /// <summary>
         /// The default recursion limit if we don't get a value from
         /// <see cref="System.Text.Json.JsonSerializerOptions.MaxDepth"/>.
         /// The service limit is (currently) 10.
         /// </summary>
-        public static int MaxJsonRecursionDepth = 64;
+        public const int MaxJsonRecursionDepth = 64;
+
+        /// <summary>
+        /// The default size of buffer to use when copying data between streams.
+        /// </summary>
+        public const int CopyBufferSize = 81920;
```

Comment:
Maybe a note as to why this value was chosen? IIRC, this is what .NET uses in various places (looks familiar, and I remember doing something similar a few years back).

Reply:
[I just stole it (and related code) from Azure.Core.](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/src/RequestContent.cs#L82)  I want to pull the Stream sync copy into shared source at some point.